plugins/techdocs/src/reader/transformers/simplifyMkdocsFooter.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Transformer } from './index';
+
+export const simplifyMkdocsFooter = (): Transformer => {
+  return dom => {
+    // Remove the header
```

Comment:
LGTM, just update or delete this comment and we can merge this PR! 🥳

Reply:
Nice catch. I've just validated your suggestion. Thanks